### PR TITLE
EGG-757: Log worker performance from Superbolt worker rake task

### PR DESCRIPTION
**Pull request by ajstiles**
[EGG-757](https://socialcoders.atlassian.net/browse/EGG-757)

### CODE REVIEW

**Comment by charliemcelfresh** [comment]
:hamburger:

File: lib/superbolt/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Superbolt
-  VERSION = "0.7.5"
+  VERSION = "0.7.6"
```
Comment (grzuy) [inline lib/superbolt/version.rb]: @ajstiles There a 0.7.6 in the queue already. Mind reviwing it? https://github.com/socialchorus/superbolt/pull/13 :-)

File: lib/tasks/superbolt.rake [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+  else
+    nil
+  end
+
   begin
     Superbolt::App.new(Superbolt.app_name, {}).run do |message, logger|
-      Superbolt::Router.new(message, logger).perform
+      begin
+        Superbolt::Router.new(message, logger).perform
+        statsd.increment("app.#{Superbolt.app_name}.messages.#{message["event"]}.success") if statsd
```
Comment (grzuy) [inline lib/tasks/superbolt.rake]: Should we name `superbolt` here?

Like for example going:

```
"superbolt.#{Superbolt.app_name}.messages.#{message['event']}.success"
```

?

Thinking what would be clearer to the user going through the metrics in the statsd UI...

**Comment by grzuy** [comment]
@ajstiles Apart from minor comments i am :+1:

File: lib/tasks/superbolt.rake (hunk 2 above)
Comment (ajstiles) [inline lib/tasks/superbolt.rake]: Great idea @grzuy.

**Pull request merged**